tools/appimage.sh

```diff
@@ -134,9 +473,10 @@ sed -i -e 's|^echo|# echo|g' usr/bin/$APP.wrapper # Make less verbose
 # Go out of AppImage
 cd ..
 
-ARCH="x86_64"
-generate_appimage
-
-cp ../out/*AppImage ..
+# NOTE: REPLACE THIS WITH A MIRRORED COPY OF APPIMAGETOOL
+wget -c https://github.com/probonopd/AppImageKit/releases/download/8/appimagetool-x86_64.AppImage
```

Comment:
Please update URL to https://psgithub.file.core.windows.net/psgithub/appimaged-x86_64.AppImage